packages/bot-web-ui/src/pages/server-bot/server-bot.scss

```diff
@@ -0,0 +1,96 @@
+.server-bot-container {
+    width: 50rem;
+}
+
+.server-bot {
+    display: flex;
+    flex-direction: row;
+    gap: 5rem;
+
+    .load-strategy__container {
+        width: 50rem;
+    }
+
+    &__title {
+        margin: 2rem;
+        text-align: center;
+
+        & button {
+            position: fixed;
+            right: 3rem;
+            background-color: var(--general-section-1);
+        }
+    }
+
+    &__image {
+        height: 45rem;
+    }
+
+    .dashboard__chart-wrapper {
+        height: 41.5rem;
+        width: calc(100vw - 55rem);
+    }
+
+    &-list {
+        margin-bottom: 2rem;
+    }
+
+    &__notifications {
+        background-color: var(--general-main-1);
+        border-radius: 1rem;
+        margin: 1rem;
+        padding: 1rem;
+        height: calc(100vh - 17rem);
+        overflow: auto;
+        width: 30rem;
+    }
+
+    &__notification {
+        margin: 1rem;
+        font-size: small;
+    }
+}
+
+.bot-list-contract__wrapper {
+    padding-bottom: 3rem;
+}
+
+.bot-list-contract {
+    &__header {
+        display: grid;
+        grid-template-columns: 1fr 1fr 2fr 1fr 1fr;
+        align-items: center;
+        column-gap: 1rem;
+        background-color: #e8e8e8;
+        border-radius: 1rem;
+        margin: 1rem;
+        padding: 1rem;
+    }
+
+    &__label:not(:last-child) {
+        width: 100%;
+        border-right: 1px solid var(--general-main-1);
+    }
+
+    &__item {
+        display: grid;
+        grid-template-columns: 1fr 1fr 2fr 1fr 1fr;
+        align-items: center;
+        justify-content: space-between;
+        column-gap: 1rem;
+        background-color: var(--general-main-1);
+        border-radius: 1rem;
+        margin: 1rem;
+        padding: 1rem;
+    }
+
+    &__actions {
+        display: flex;
+        gap: 2rem;
+    }
+
+}
+        .bot-list-btn--disabled {
+            pointer-events: none !important;
+            cursor: none !important;
+        }
```

Comment:
Please fix EOL error

Reply:
we have done it temporarily till the moment when the `bot_pause` API call will be available/ provided by BE, for now, we installed a temp plug